<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericEventParamsWidget</class>
 <widget class="QWidget" name="GenericEventParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>190</height>
   </rect>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhUppercaseOnly</set>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="sourceVerticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="sourceLabel">
       <property name="text">
        <string>Source contains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceLineEdit">
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="captionVerticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="captionLabel">
       <property name="text">
        <string>Caption contains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="captionLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="descriptionVerticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="destinationLabel">
       <property name="text">
        <string>Description contains</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Keywords separated by space</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="QLabel" name="hintLabel">
       <property name="text">
        <string>To generate Generic Event, please refer to &lt;a href=&quot;https://support.networkoptix.com/hc/en-us/articles/219573367-Nx-Server-HTTP-REST-API&quot;&gt;Server&amp;nbsp;API&lt;/a&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextBrowserInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
